node/cn/gasprice/feehistory_test.go

```diff
@@ -54,15 +56,17 @@ func TestFeeHistory(t *testing.T) {
 		{1000, 1000, 2, rpc.PendingBlockNumber, nil, 32, 1, nil},
 	}
 	magmaBlock, kaiaBlock := int64(16), int64(20)
-	backend, gov := newTestBackend(t, big.NewInt(magmaBlock), big.NewInt(kaiaBlock))
+	backend, govModule := newTestBackend(t, big.NewInt(magmaBlock), big.NewInt(kaiaBlock))
+	// govModule.EXPECT().EffectiveParamSet(gomock.Any()).Return(gov.ParamSet{UnitPrice: backend.ChainConfig().UnitPrice}).AnyTimes()
```

Comment:
It seems this commented line can be deleted.

Reply:
applied 8b08c5d0